docs/design_proposals/pixi_global_manifest.md

```diff
@@ -294,3 +294,17 @@ In order to make it easier to manage manifests in version control, we could allo
 ``` title="config.toml"
 global_manifests = "/path/to/your/manifests"
 ```
+
+
+### No activation
+
+The current `pixi global install` features `--no-activation`.
+When this flag is set `CONDA_PREFIX` and `PATH` will not be set in the activation script.
```

Comment:
```suggestion
When this flag is set, `CONDA_PREFIX` and `PATH` will not be set when running the exposed binary.
```
The activation script is an internal term. Also it is a bit confusing because conda envs also have activation scripts. Here, the doc talks mainly about exposed binaries.